docs/variables.md

```diff
@@ -33,6 +33,7 @@ The following variables are made available to reference various metadata of a wo
 | Variable | Description|
 |----------|------------|
 | `pod.name` | Pod name of the container/script |
+| `pod.retryAttempt` | The retry attempt number of the container/script if retryStrategy is specified |
```

Comment:
I think just `retryAttempt` as in `{{retryAttempt}}` is fine. We use the parameter-only naming convention when doing self-reference elsewhere. Furthermore, a "retry attempt" doesn't have to do with a `Pod` at all.

Reply:
updated in new commit